implementations/rust/ockam/ockam_entity/src/credential/messages.rs

```diff
@@ -0,0 +1,15 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+pub enum CredentialProtocolMessage {
+    IssueOfferRequest(String),
+    IssueOffer(crate::CredentialOffer),
+    IssueRequest(crate::CredentialRequest, Vec<crate::CredentialAttribute>),
+    IssueResponse(crate::CredentialFragment2),
+    PresentationOffer,
+    PresentationRequest(crate::ProofRequestId),
+    PresentationResponse(crate::CredentialPresentation),
```

Comment:
Can we `use` instead of prefix, or is there an advantage here?

Reply:
There were duplicates in names, but I'll check again